python/tox.ini

```diff
@@ -10,15 +10,17 @@ envlist =
 commands = 
     pytest -rsXx {posargs:tests}
     pip wheel . -w {toxworkdir}/wheelhouse
- 
 passenv = 
     TRAVIS 
-    LD_LIBRARY_PATH 
-    CFLAGS
+    DCC_RS_DEV
 deps = 
     pytest
     pytest-faulthandler
     pdbpp
+    cffi
+    requests
+    attrs
+    six
```

Comment:
aren't these deps installed with the "deltachat" package?  seems redundant here.

Reply:
You're right (TIL though), this was from earlier flailing about desperately trying to get things to work